docs/syntax/pipes.md

```diff
@@ -19,59 +20,58 @@ default Vento has the `escape` and `unescape` filter out of the box, to escape
 and unescape HTML code:
 
 ```vento
-{{ "<h1>Hello world</h1>" |> escape }}
+{{ "<h1>Hello, world!</h1>" |> escape }}
 ```
 
 If the filter accepts additional arguments you can pass them between
 parenthesis:
 
 ```vento
-{{ "<h1>Hello world</h1>" |> filter_name(arg1, arg2) }}
+{{ "<h1>Hello, world!</h1>" |> filter_name(arg1, arg2, ...) }}
 ```
 
 ## Global functions
 
-If the function is a standard function available globally (for example
-`JSON.stringify()`), Vento will execute:
+Vento will execute functions available in the standard JavaScript namespace, such as `JSON.stringify()`:
```

Comment:
I like this wording better than before but let me know your thoughts...

Reply:
I'm not a native English speaker (as you can guess), so if you say it's better, I'm okay :).